res/css/views/elements/BlockedIcon.pcss

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.mx_BlockedIcon {
+    background-color: #f07a12;
```

Comment:
Would be nice to reuse colors, like --var(warning) or something.
Not a blocker, if it's not a 5 minute task then let's file an issue instead

Reply:
Totally agree. Unfortunately, when not using a custom theme we don't have this variables. This is a > 5 minutes task.

Reply:
WIth Angelo, we want to use a different way to custom the theme : https://github.com/tchapgouv/tchap-web-v4/issues/130 
It could be add to the issue when it will be priorized.